pkg/ddevapp/craftcms.go

```diff
@@ -124,9 +134,11 @@ func craftCmsPostStartAction(app *DdevApp) error {
 			"CRAFT_DB_DATABASE":     "db",
 			"CRAFT_DB_USER":         "db",
 			"CRAFT_DB_PASSWORD":     "db",
+			"CRAFT_WEB_URL":         app.GetPrimaryURL(),
+			"CRAFT_WEB_ROOT":        app.GetAbsDocroot(true),
 			"MAILHOG_SMTP_HOSTNAME": "127.0.0.1",
 			"MAILHOG_SMTP_PORT":     "1025",
-			"PRIMARY_SITE_URL":      app.GetPrimaryURL(),
+			"PRIMARY_SITE_URL":      app.GetPrimaryURL(), // for backward compatibility only
```

Comment:
I thought the decision in https://github.com/ddev/ddev/issues/4650 was not to include this.

Reply:
Removal is breaking and imho we should give the users some time to use the new config.

Reply:
I think this would only affect new projects.